Gordon360/Services/RecIM/ParticipantService.cs

```diff
@@ -0,0 +1,242 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Models.CCT.Context;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Data.SqlClient;
+
+namespace Gordon360.Services.RecIM
+{
+    public class ParticipantService : IParticipantService
+    {
+        private readonly CCTContext _context;
+        private readonly IAccountService _accountService;
+
+        public ParticipantService(CCTContext context, IAccountService accountService)
+        {
+            _accountService = accountService;
+            _context = context;
+        }
+        public IEnumerable<LookupViewModel> GetParticipantLookup(string type)
+        {
+            if (type == "status")
+            {
+                var res = _context.ParticipantStatus
+                            .Select(s => new LookupViewModel
+                            {
+                                ID = s.ID,
+                                Description = s.Description
+                            })
+                            .AsEnumerable();
+                return res;
+            }
+            if (type == "activitypriv")
+            {
+                var res = _context.PrivType
+                            .Select(s => new LookupViewModel
+                            {
+                                ID = s.ID,
+                                Description = s.Description
+                            })
+                            .AsEnumerable();
```

Comment:
These could also be incorporated into a single 'code table'. It would be much cleaner.

Reply:
refer to above conversation for result